docs/sdks/server-sdks/dotnet/bandits.mdx

```diff
@@ -0,0 +1,209 @@
+---
+title: Contextual Bandits
+sidebar_position: 5
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ApiOptionRef from '@site/src/components/ApiOptionRef';
+
+## Usage with Contextual Multi-Armed Bandits
+
+Eppo also supports contextual multi-armed bandits. You can read more about them in the [high-level documentation](/contextual-bandits).
+Bandit flag configuration--including setting up the flag key, status quo variation, bandit variation, and targeting rules--are configured within
```

Comment:
I think an emdash is the correct punctuation here
ex: 
`Bandit flag configuration &mdash; including setting up the flag key`

Reply:
I'll be honest, I think I screwed up formatting an unordered list there somehow. This information works a lot better as an unordered list than as a single sentence.